.github/dependabot.yml

```diff
@@ -3,7 +3,7 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: daily
+      interval: weekly
```

Comment:
It's nice to have a different schedule for the actions so there's less spam on Mondays. I'd keep it as daily.

Reply:
```suggestion
      interval: daily
```